lib/common-yargs.js

```diff
@@ -44,7 +44,6 @@ const withRepo = yargs => yargs.option("repo", {
 	alias: "r",
 	describe: "Repository",
 	type: "string",
-	demandOption: true,
```

Comment:
What's the implication of this on other commands? Will they still be validating correctly as a result of this?